docs/how-to/h-deploy-azure.md

```diff
@@ -0,0 +1,250 @@
+
+```shell
+To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code <YOUR_CODE> to authenticate.
+```
+
+In the browser, open the [authentication page](https://microsoft.com/devicelogin) and enter the code `<YOUR_CODE>` provided in the output. 
+
+You will be asked to authenticate twice, for allowing the creation of two different resources in Azure.
+
+You will see a message that the credentials have been added locally:
+
+```shell
+Credential <CREDENTIAL_NAME> added locally for cloud "azure".
+```
+
+Once the credentials are correctly added, we can bootstrap a controller:
+
+```shell
+juju bootstrap azure <CONTROLLER_NAME>
+```
+
+## Deploy charms
+
+Create a new Juju model, if needed:
+
+```shell
+juju add-model <MODEL_NAME>
+```
+
+[note type="caution"]
+(Optional) Increase the debug level if you are troubleshooting charms:
+```shell
+juju model-config logging-config='<root>=INFO;unit=DEBUG'
+```
+[/note]
+
+Deploy and integrate Kafka and ZooKeeper:
+
+```shell
+juju deploy zookeeper -n3 --channel 3/stable [--constraints "instance-type=<INSTANCE_TYPE>"]
+juju deploy kafka -n3 --channel 3/stable [--constraints "instance-type=<INSTANCE_TYPE>"]
+juju integrate kafka zookeeper
+```
+
+[note type="caution"]
+Note that the smallest instance types on Azure may not have enough resources for hosting 
+a Kafka broker. We recommend selecting an instance type that provides at the very least `8` GB of RAM and `4` cores, e.g. `Standard_A4_v2`.
+For more guidance on production environment sizing, see the [Requirements page](/t/charmed-kafka-reference-requirements/10563).
+You can find more information about the available instance types in the [Azure documentation](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/overview).
+[/note]
+
+We also recommend to deploy a [Data Integrator](https://charmhub.io/data-integrator) for creating an admin user to manage the content of the Kafka cluster:
+
+```shell
+juju deploy data-integrator admin --channel edge \
+  --config extra-user-roles=admin \
+  --config topic-name=admin-topic
+```
+
+And integrate it with the Kafka application:
+
+```shell
+juju integrate kafka admin
+```
+
+For more information on Data Integrator and how to use it, please refer to the [how-to manage applications](/t/charmed-kafka-how-to-manage-app/10285) guide.
+
+## Clean up
+
+[note type="caution"]
+Always clean Azure resources that are no longer necessary! Abandoned resources are tricky to detect and they can become expensive over time.
+[/note]
+
+To destroy the Juju controller and remove the Azure instance (Warning: all your data will be permanently removed):
+
+```shell
+juju destroy-controller <CONTROLLER_NAME> --destroy-all-models --destroy-storage --force
+```
+
+> Use `juju list-controllers` to retrieve the names of the controllers that have been registered to your local client. 
```

Comment:
This can be an ordinary text or a `[note]`, rather than a caution type note. Otherwise, it would seem to be a bit too repetitive.